be/src/agent/agent_server.cpp

```diff
@@ -120,14 +120,14 @@ void AgentServer::submit_tasks(TAgentResult& agent_result,
         TTaskType::type task_type = task.task_type;
         int64_t signature = task.signature;
 
-#define HANDLE_TYPE(t_task_type, work_pool, req_member)                         \
-    case t_task_type:                                                           \
-        if (task.__isset.req_member) {                                          \
-            work_pool->submit_task(task);                                       \
-        } else {                                                                \
-            ret_st = Status::InvalidArgument(strings::Substitute(               \
-                    "task(signature=$0) has wrong request member", signature)); \
-        }                                                                       \
+#define HANDLE_TYPE(t_task_type, work_pool, req_member)                                     \
+    case t_task_type:                                                                       \
+        if (task.__isset.req_member) {                                                      \
+            work_pool->submit_task(task);                                                   \
+        } else {                                                                            \
+            ret_st = Status::InvalidArgument("task(signature={}) has wrong request member", \
```

Comment:
use #req_member to print the real request member, such as 'create_tablet_req', 'drop_tablet_req', and so on, otherwise this error message is still confusing.

Reply:
Change it like this? : `Status::InvalidArgument("task(signature={}, request menmber={}) has wrong request member", signature, req_menber);`

Reply:
#req_member

like: task(signature={}) missing request member={}

Reply:
ok